src/ot2rec_report/templates/report_excludebadtilts.ipynb

```diff
@@ -33,7 +38,7 @@
     "ebt_config_yaml = f\"{proj_name}_exclude_bad_tilts.yaml\"\n",
     "\n",
     "if os.path.isfile(ebt_config_yaml) is False:\n",
-    "    raise IOError(f\"Exclude bad tilts yaml file {ebt_config_yaml} not found.\")\n",
+    "    print(f\"Exclude bad tilts yaml file {ebt_config_yaml} not found.\")\n",
```

Comment:
Why not `raise FileNotFoundError`?

Reply:
Good idea - will put that in (new PR incoming with the PyPI stuff)